packages/builder/src/types.ts

```diff
@@ -207,6 +207,15 @@ export interface TaskDefinition extends ConfigurableTaskDefinition {
   readonly positionalParamDefinitions: ParamDefinitionAny[]
 }
 
+export interface ScriptCheckpoint {
+  [network: string]: ScriptNetCheckpoint
+};
+
+export interface ScriptNetCheckpoint {
+  timestamp: number
+  metadata: {[key: string]: string}
+};
```

Comment:
No need to create `interfaces`. Let's define a `class` then, in typescripts, this will automatically becomes a type

Reply:
This one can have a class but the main type can't